model_info/bnwiki.reverted.md

```diff
@@ -1,12 +1,12 @@
 Model Information:
```

Comment:
Tangentially, in future work let's include filename and checksum of input data sets in model info.